### PR TITLE
Fix intermittent panic in notify issue change content

### PR DESCRIPTION
**Pull request by zeripath**
Ensure that issue pullrequests are loaded before trying to set the self-reference.

### CODE REVIEW

**Review by KN4CK3R — APPROVED** [review]

File: services/webhook/notifier.go [hunk 1]
```diff
@@ -196,7 +195,6 @@ func (m *webhookNotifier) NotifyIssueChangeTitle(ctx context.Context, doer *user
 			log.Error("LoadPullRequest failed: %v", err)
 			return
 		}
-		issue.PullRequest.Issue = issue
```
Comment (delvh) [inline services/webhook/notifier.go:199]: ?

File: services/webhook/notifier.go [hunk 2]
```diff
@@ -150,7 +150,6 @@ func (m *webhookNotifier) NotifyIssueChangeAssignee(ctx context.Context, doer *u
...
```
Comment (delvh) [inline services/webhook/notifier.go:153]: ?

**Review by delvh — COMMENTED** [review]
I don't quite understand the changes in this PR:
1. Why was the indentation in `LoadPullRequest` increased? It doesn't change any logic…
2. Why were the two unrelated lines in `services/webhook/notifier.go` removed?

**Comment by KN4CK3R** [comment]
LoadPullRequest does the assignment already.

**Review by delvh — APPROVED** [review]

**Comment by lunny** [comment]
Please send backport

**Comment by yardenshoham** [comment]
Please send backport

**Pull request merged**